src/Microsoft.Health.Fhir.Api/Configs/FeatureConfiguration.cs

```diff
@@ -34,5 +34,10 @@ public class FeatureConfiguration
         /// Gets or sets a value indicating whether Validate is enabled or not.
         /// </summary>
         public bool SupportsValidate { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether anonymized export is enabled or not.
+        /// </summary>
+        public bool SupportAnonymizedExport { get; set; }
```

Comment:
Should this be an option of the ExportJobConfiguration?

Reply:
We would prefer default support anonymized export in FHIR server, so we do not want to make it as a settings for export. We put the flag here is to make sure it would not enable by default during preview stage and not want to enable it on PaaS automatically after OSS nuegt updated.